packages/playground/_stories/main/Toolbar/Toolbar.stories.ts

```diff
@@ -0,0 +1,115 @@
+import { html } from "lit";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { unsafeHTML } from "lit/directives/unsafe-html.js";
+import type { StoryFn, Meta } from "@storybook/web-components";
+
+import type Toolbar from "@ui5/webcomponents/dist/Toolbar";
+
+import argTypes, { componentInfo } from "./argTypes.js";
+import type { StoryArgsSlots } from "./argTypes.js";
+import type { UI5StoryArgs } from "../../../types.js";
+
+import { DocsPage } from "../../../.storybook/docs";
+
+const component = "ui5-toolbar";
+
+export default {
+  title: "Main/Toolbar",
+  component,
+  subcomponents: { ToolbarItem: "ui5-toolbar-item" },
+  argTypes,
+  parameters: {
+    docs: {
+      page: DocsPage({ ...componentInfo, component })
+    },
+  },
+} as Meta<Toolbar>;
+
+const Template: UI5StoryArgs<Toolbar, StoryArgsSlots> = (args) => {
+  return html`<ui5-toolbar design="Solid" styling="${ifDefined(args.styling)}">
+        <ui5-toolbar-button icon="add" text="Left 1 (long)" width="150px" design="Default"></ui5-toolbar-button>
+				<ui5-toolbar-button icon="decline" text="Left 2"></ui5-toolbar-button>
```

Comment:
Can we make these stories more dynamic using template literals for all toolbar properties and the default slot.

```js
return html` <ui5-toolbar
    design="${ifDefined(args.design)}"
    styling="${ifDefined(args.styling)}"
    ?disabled="${ifDefined(args.disabled)}"
    ?reverse-overflow="${ifDefined(args.reverseOverflow)}"
    .....
>
    ${unsafeHTML(args.default)}
</ui5-toolbar>`;

.....

Basic.args = {
  design: ToolbarDesign.Solid,
  default: `
<ui5-toolbar-button icon="decline" text="Mid 2"></ui5-toolbar-button>
<ui5-toolbar-button icon="add" text="Right 1"></ui5-toolbar-button>
<ui5-toolbar-button icon="employee" text="Right 4"></ui5-toolbar-button>
<ui5-toolbar-button id="myOverflowBtn" icon="employee" text="Call me later"></ui5-toolbar-button>`,
};